src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -11,12 +11,14 @@ namespace System.IO
     public class FileSystemAclExtensionsTests
     {
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot)]
```

Comment:
It looks like you're disabling every test in this class?  If so, it'd be better to put the SkipOnTargetFramework attribute once on the test class rather than on each method.